app/jobs/order_cycle_notification_job.rb

```diff
@@ -0,0 +1,8 @@
+
+OrderCycleNotificationJob = Struct.new(:order_cycle) do
+  def perform
+    @suppliers = order_cycle.suppliers
```

Comment:
There's no need to use an instance variable `@suppliers` here. A local variable would be more suitable since we don't need the var to be accessible by the class, but given the simplicity I would probably just do this on one line without using a variable.